modoboa_pdfcredentials/documents.py

```diff
@@ -44,37 +56,82 @@ def page_template(canvas, doc):
         canvas.setCreator("Modoboa")
         footer = [Paragraph(_("Powered by Modoboa - Mail hosting made simple"),
                             styles["Footer"])]
-        Frame(0, 0, 21 * cm, 4 * cm).addFromList(footer, canvas)
+        Frame(0, 0, 21 * cm, 2 * cm).addFromList(footer, canvas)
 
     filename = lib.get_creds_filename(account)
     buff = BytesIO()
     doc = SimpleDocTemplate(buff, pagesize=A4)
+
     story = []
-    story.append(resized_image(lib.get_document_logo(), 6*cm))
+
+    story.append(resized_image(lib.get_document_logo(), 8*cm))
+
     story.append(Spacer(1, 1 * cm))
     story.append(Paragraph(_("Personal account information"), styles["Title"]))
-    story.append(Spacer(1, 2 * cm))
+
+    story.append(Spacer(1, 1 * cm))
     story.append(Paragraph(_("""
 Dear %s, this document contains the credentials you will need
 to connect to Modoboa. Learn the content and destroy
 the document as soon as possible.
 """) % account.fullname, styles["Normal"]))
-    url = "http(s)://{}{}".format(
-        Site.objects.get_current().domain, settings.LOGIN_URL)
+
+    story.append(Spacer(1, 0.2 * cm))
+    story.append(Paragraph(_("Webinterface:"), styles["h3"]))
+
+    url = "https://{}".format(
+        Site.objects.get_current().domain)
     data = [
         ["URL", url],
         [_("Username"), account.username],
         [_("Password"), password]
     ]
     table = Table(data)
     table.setStyle(TableStyle([
+        ('TEXTCOLOR', (1, 0), (1, 0), colors.blue),
+        ('GRID', (0, 0), (-1, -1), 1, colors.black),
+        ('BACKGROUND', (0, 0), (0, -1), colors.lightgrey),
+    ]))
+    story.append(table)
+
+    story.append(Spacer(1, 0.5 * cm))
+    story.append(Paragraph(_("""
+Here you can view your emails anytime online and check the spam filter and settings.
+"""), styles["Normal"]))
+
+    story.append(Spacer(1, 0.2 * cm))
+    story.append(Paragraph(_("Please change your password now!"), styles["Warning"]))
+
+    story.append(Spacer(1, 1 * cm))
+    story.append(Paragraph(_("Installation PC/Tablet/Smartphone:"), styles["h3"]))
+
+    story.append(Spacer(1, 0.2 * cm))
+    data = [
+        [_("Username"), account.username],
+        [_("Password"), password],
+        ["Type", "IMAP"],
+        ["IMAP-Server (Inbox)", account.username.split("@")[1] ],
+        ["Security", "SSL"],
+        ["Port", "993"],
+        ["SMTP-Server (Outbox)", account.username.split("@")[1] ],
+        ["Security", "TLS"],
+        ["Port", "587"],
+        ["Requires Auth?", "Yes (use same as Inbox)"],
+    ]
```

Comment:
Hardcoding connection settings is a bad idea... I know the installer deploys postfix and dovecot using those settings by default but it is not the only way of installation. People can also change those parameters after...